ref/general/hardening-intro.adoc

```diff
@@ -22,49 +22,38 @@ Hardening is a general term that is used to describe the measures that are taken
+* *Carefully configure file permissions*: When you configure file permissions, ensure that the users who want access to the configuration fall into the appropriate permission bit. Users that are not the owner and are not part of the authorized group fall into the `other` permission bit, which is marked as `0`.
+* *Encrypt passwords*: Use encryption for passwords instead of Base64 encoding. You can use the `securityUtility` command, which provides plain text encryption for Open Liberty. For more information about the `securityUtility command`, go to <link pending>.
 
 
 [#network]
 == Network
-Network intrusions occur when unauthorized users monitor or alter network traffic. Replay attacks and man-in-the-middle (MITM) attacks are two examples of network intrusions.
-
-To learn more about network vulnerabilities and ways to prevent network intrusions, go to <link to network topic>.
-
-=== Best practices
+Network intrusions occur when unauthorized users monitor or alter network traffic. Replay attacks and man-in-the-middle (MITM) attacks are two examples of types of network intrusions.
 
-===== Use a firewall
-To prevent network intrusions, use a firewall to shield production Open Liberty servers so that they don't run in the de-militarized zone (DMZ).
+To learn more about network vulnerabilities and ways to prevent network intrusions, go to <link pending>.
 
-===== Use Transport Security
-It's important to use the feature:transportSecurity-1.0[] feature to ___. This feature enables the feature:ssl-1.0[] feature In addition to enabling `transportSecurity-1.0`, disable non-SSL ports by setting them to `-1` in the `server.xml` file. Only use certificates that are signed by a Certificate Authority for public-facing production servers.
+==== Best practices
 
-===== alsdkfj;asdf
-Configure applications to run on a virtual host to keep admin traffic separate from application requests.
+* *Use a firewall*: Use a firewall to shield production Open Liberty servers so that they don't run in the DMZ. The DMZ is an unsecured network segment that exists between internal and external networks. Firewalls help prevent security intrusions by monitoring network traffic and blocking specific traffic based on security rules.
+* *Enable the `transportSecurity` feature*: The feature:transportSecurity-1.0[] feature enables the feature:ssl-1.0[] feature to support SSL connections and adds the ability to configure outbound transports. In addition to enabling `transportSecurity-1.0`, disable non-SSL ports by setting them to `-1` in the `server.xml` file, and use only certificates that are signed by a certificate authority for public-facing production servers.
```

Comment:
do we want to include version #s on features? Does the macro always grab the most current version? This should be consistent across the docs, whatever the call

Reply:
I'll _try_ to update it so that it resolves to the latest version, need to think about this in the context of Antora transition too